documenting.rst

```diff
@@ -1792,10 +1792,7 @@ Ask on doc-sig, or better, make a PR on the `devguide
 I have a translation, but not on git, what should I do?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Just ask for help on the doc-sig mailing list and our python-fu, git-fu
-and bash-fu combined will help you create an appropriate repository. If
-you use a tool like transifex don’t worry.  Keeping them in sync is not
-that hard.
+You can ask for help on the doc-sig mailing list, and the team will help you create an appropriate repository. You can still use tools like transifex, if you like.
```

Comment:
I think the new text sounds much better than the existing one. Thank you!

My only suggestion is to keep the text max 80 characters each line.

```suggestion
You can ask for help on the doc-sig mailing list, and the team will help you
create an appropriate repository. You can still use tools like transifex,
if you like.
```